core-bundle/src/Controller/AbstractController.php

```diff
@@ -14,15 +14,17 @@
 
 use Contao\CoreBundle\Framework\ContaoFramework;
 use FOS\HttpCacheBundle\Http\SymfonyResponseTagger;
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController as SymfonyAbstractController;
 
 abstract class AbstractController extends SymfonyAbstractController
 {
-    public static function getSubscribedServices()
+    public static function getSubscribedServices(): array
```

Comment:
As already mentioned in another PR (as a reminder): inheriting classes then have to implement the strict method signature, which is a BC break. Also, the method `getSubscribedServices` is defined per EventSubscriberInterface, so this method differs from the interface (which is technically irrelevant because it's compatible).